sdk/eventhubs/microsoft-azure-eventhubs-perf/src/main/java/com/microsoft/azure/eventhubs/perf/SendEventBatchPartitionTest.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.eventhubs.perf;
+
+import com.microsoft.azure.eventhubs.BatchOptions;
+import com.microsoft.azure.eventhubs.EventData;
+import com.microsoft.azure.eventhubs.EventDataBatch;
+import com.microsoft.azure.eventhubs.PartitionSender;
+import com.microsoft.azure.eventhubs.perf.core.EventHubsPerfStressOptions;
+import com.microsoft.azure.eventhubs.perf.core.ServiceTest;
+import reactor.core.publisher.Mono;
+
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Runs the Send Events Batch Performance Test for EventHubs.
+ */
+public class SendEventBatchPartitionTest extends ServiceTest<EventHubsPerfStressOptions> {
```

Comment:
Do we need separate classes for testing "send to specific partition" vs "send to any partition"?